ocp_resources/datavolume.py

```diff
@@ -90,6 +90,7 @@ def __init__(
         privileged_client=None,
         yaml_file=None,
         delete_timeout=TIMEOUT_4MINUTES,
+        use_storage_api=False,
```

Comment:
```suggestion
        storage_api="pvc",
```
I prefer to see here `pvc` as default, and add `TODO` to switch to `storage` when needed or remove it and use only `storage`

Reply:
Ack